src/components/Change/ChangeSummary.vue

```diff
@@ -46,7 +46,7 @@
     <!-- Business Addresses -->
     <template v-if="haveOfficeAddressesChanged">
       <v-divider class="mx-8" />
-      <article id="address-summary-section" class="section-container">
+      <article id="address-summary-section">
```

Comment:
After:

![image](https://user-images.githubusercontent.com/10002889/232634395-e9c3ba70-3bef-4a64-94d3-969c1e0b51a6.png)

Reply:
The OfficeAddress component has a section-container class already -- needed so we could just drop it in the YourCompany wrapper. So that class has to be removed here.

This is a special case -- there are no other components that we can both drop in the wrapper AND is on the Summary pages.